### PR TITLE
Remove friction from SIG Charter process through better documentation.

### PR DESCRIPTION
**Pull request by pwittrock**

### CODE REVIEW

File: committee-steering/governance/README.md [hunk 1]
```diff
@@ -1,23 +1,26 @@
...
-template documents for SIGs to adapt.  The goals are to define the baseline needs for SIGs to self govern
-and organize in a way that addresses the needs of the core Kubernetes project.
-
-The documents are focused on:
+## Steps to create a SIG charter
 
-- Outlining organizational responsibilities
-- Outlining organizational roles
-- Outlining processes and tools
+1. Copy the template into a new file under community/sig-*YOURSIG*/charter ([sig-architecture example])
```
Comment (pmorie) [inline committee-steering/governance/README.md]: Should this be `charter.md` ?

File: committee-steering/governance/README.md [hunk 2]
```diff
@@ -1,23 +1,26 @@
...
+2. Read the [Recommendations and requirements] so you have context for the template
+3. Customize your copy of the template for your SIG.  Feel free to make adjustments as needed.
+4. Update [sigs.yaml] with the individuals holding the roles as defined in the template.
```
Comment (pmorie) [inline committee-steering/governance/README.md]: Completely off topic, I'm astounded that this link to sigs yaml works. HOW DOES THIS WORK

File: committee-steering/governance/README.md [hunk 3]
```diff
@@ -1,23 +1,26 @@
-# SIG Governance Template
+# SIG Charter Guide
```
Comment (pmorie) [inline committee-steering/governance/README.md]: A link to the requirement that SIGs should have a charter would be extremely helpful here to contextualize what the purpose of the rest of the doc is.

File: committee-steering/governance/README.md (hunk 2 above)
Comment (pwittrock) [reply committee-steering/governance/README.md]: Look at the bottom of the page

File: committee-steering/governance/README.md (hunk 3 above)
Comment (pwittrock) [reply committee-steering/governance/README.md]: Added a short section about this at the top.

File: committee-steering/governance/README.md (hunk 1 above)
Comment (pwittrock) [reply committee-steering/governance/README.md]: done

**Comment by pwittrock** [comment]
PTAL

**Comment by pmorie** [comment]
This looks like a constructive change, thanks a lot @pwittrock

**Pull request merged**